.github/workflows/release.yml

```
@@ -0,0 +1,413 @@
+        with:
+          fetch-depth: 0
+          
+      - uses: jdx/mise-action@v2
+          
+      - name: Get current version
+        id: version
+        run: |
+          # Get the latest tag
+          CURRENT_VERSION=$(git describe --tags --abbrev=0 2>/dev/null || echo "v0.0.0")
+          echo "current_version=${CURRENT_VERSION}" >> $GITHUB_OUTPUT
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)